cypress/support/fragments/data_import/action_profiles/actionProfiles.js

```diff
@@ -0,0 +1,22 @@
+import { getLongDelay } from '../../../utils/cypressTools';
+import { Button } from '../../../../../interactors';
+
+export default class ActionProfiles {
```

Comment:
Probably let's not use classes and rather follow our common approach with default export?